plugins/orchestrator/src/components/WorkflowDefinitionsListComponent/WorkflowDefinitionsListComponent.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Table, TableColumn } from '@backstage/core-components';
+import { useApi, useRouteRef } from '@backstage/core-plugin-api';
+
+import DeleteForever from '@material-ui/icons/DeleteForever';
+import Edit from '@material-ui/icons/Edit';
+import Pageview from '@material-ui/icons/Pageview';
+import PlayArrow from '@material-ui/icons/PlayArrow';
+
+import {
+  extractWorkflowFormatFromUri,
+  WorkflowItem,
+} from '@janus-idp/backstage-plugin-orchestrator-common';
+
+import { orchestratorApiRef } from '../../api';
+import {
+  editWorkflowRouteRef,
+  executeWorkflowRouteRef,
+  workflowDefinitionsRouteRef,
+} from '../../routes';
+
+type WorkflowsTableProps = {
+  items: WorkflowItem[];
+};
+
+export const WorkflowsTable = ({ items }: WorkflowsTableProps) => {
+  const orchestratorApi = useApi(orchestratorApiRef);
+
+  const navigate = useNavigate();
+  const definitionLink = useRouteRef(workflowDefinitionsRouteRef);
+  const executeWorkflowLink = useRouteRef(executeWorkflowRouteRef);
+  const editLink = useRouteRef(editWorkflowRouteRef);
+
+  interface Row {
+    id: string;
+    name: string;
+    format: string;
+  }
+
+  const columns: TableColumn[] = [{ title: 'Name', field: 'name' }];
+  const data: Row[] = items.map(item => {
+    return {
+      id: item.definition.id,
+      name: item.definition.name ?? '',
+      format: extractWorkflowFormatFromUri(item.uri),
+    };
+  });
+
+  const doView = useCallback(
+    (rowData: Row) => {
```

Comment:
redundant `useCallback`

Reply:
There is a refactoring going on and this component will likely be replaced with new code.